101-rds-deployment-full-ha/azuredeploy.json

```diff
@@ -0,0 +1,1180 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectName": {
+            "type": "string",
+            "defaultValue": "GEN-UNIQUE",

```

Comment:
this is not valid in a template, only param files... if this needs to be globally unique, then use an expressing with uniqueString() to generate a globally unique default.